api/src/main/java/io/strimzi/api/kafka/model/connector/AlterOffsets.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.connector;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.common.Constants;
+import io.strimzi.api.kafka.model.common.ResourceReference;
+import io.strimzi.api.kafka.model.common.UnknownPropertyPreserving;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Configuration for altering connector offsets.
+ */
+@Buildable(
+        editableEnabled = false,
+        builderPackage = Constants.FABRIC8_KUBERNETES_API
+)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@JsonPropertyOrder({"fromConfigMap"})
+@EqualsAndHashCode
+@ToString
+public class AlterOffsets implements UnknownPropertyPreserving {
+    private ResourceReference configMapReference;
```

Comment:
Same as below, why don't you name it Why don't you just name it `fromConfigMap` / `getFromConfigMap` / `setFromConfigMap`?

Reply:
I've updated it as you suggested